cloud/aws/services/ec2/instances.go

```diff
@@ -69,8 +72,32 @@ func (s *Service) InstanceIfExists(instanceID *string) (*Instance, error) {
 }
 
 // CreateInstance runs an ec2 instance.
-func (s *Service) CreateInstance(machine *clusterv1.Machine) (*Instance, error) {
-	input := &ec2.RunInstancesInput{}
+func (s *Service) CreateInstance(machine *clusterv1.Machine, config *v1alpha1.AWSMachineProviderConfig, clusterStatus *v1alpha1.AWSClusterProviderStatus) (*Instance, error) {
+	id := config.AMI.ID
+	if id == nil {
+		// TODO(chuckha) region is defined by the session which is bad for HA stuff
```

Comment:
Arrgh, we need to bubble this up to a tracking issue, I think. Ideally, we'd be able to define the region on a per-cluster level, though some might also want to specify it at the machine level.

Reply:
do you think it's ok to defer machine-level-region feature for later or is this a requirement for MVP?
Regardless, issue is being tracked here: https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/144

Reply:
I definitely don't think machine-level override is a requirement for MVP, I think cluster-level region is borderline at this point.